application-licensing-licensor/application-licensing-licensor-ui/src/main/resources/Licenses/Code/Translations.xml

```diff
@@ -83,6 +87,15 @@ licensor.licenseManager.heading=Licensed Extensions
 licensor.licenseManager.hint=Here's a list of extensions installed in your wiki that require a license to be activated. You can buy a license or get a 10-day trial license that can be extended only once with 10 days.
 licensor.missingLicenses=The following extensions do not have a valid license. Please visit the {0}Licenses section{1}.
 licensor.missingLicense=There is no valid license for {0}. Please visit the {1}Licenses section{2}.
+licensor.notification.done=The following extension have been upgraded to the last compatible version: {0}
+licensor.notification.failed=The following extension failed to be upgraded to the last compatible version: {0}
+licensor.notifications.event.done=Upgrade done for {0} from version {1} to {2}
+licensor.notifications.event.failed=Error while upgrading {0} from version {1} to {2}
+licensor.notifications.event.type=Automatic upgrades
+licensor.notifications.event.type.description=An extension is upgraded automatically
+licensor.notifications.event.failed.type.description=An extension fails to be upgraded automatically
+licensor.notifications.event.singular=Upgrade to the last compatible version has been attempted for an extension
+licensor.notifications.event.plural=Upgrade to the last compatible version has been attempted for multiple extensions
```

Comment:
I find the above keys to be too generic for the values they have. As if the licensor can produce only one type of notification, related to automatic upgrades. Maybe ``licensor.notification.autoUpgrade.*`` ?